modules/appeals_api/app/workers/appeals_api/hlr_pdf_submit_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'appeals_api/hlr_pdf_constructor'
+
+module AppealsApi
+  class HlrPdfSubmitJob
+    include Sidekiq::Worker
+
+    def perform(higher_level_review_id)
+      pdf_constructor = AppealsApi::HlrPdfConstructor.new(higher_level_review_id)
+      pdf_constructor.fill_pdf
+      # set status to pending upload
+      HigherLevelReview.update(higher_level_review_id, status: 'processing')
```

Comment:
is this comment irrelevant now? (now that we're using 'processing')

Reply:
yea, I need to update the comment good catch, I was putting in the comments to mark the points until we got the central mail piece done

Reply:
fixed